atomistics/calculators/lammps/__init__.py

```
@@ -23,7 +25,11 @@
     __all__ = [
         "calc_molecular_dynamics_phonons_with_lammpslib",
     ]
-except ImportError:
+except ImportError as e:
+    warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)